### PR TITLE
Remove support for JRuby 1.7 compatibility

### PR DESCRIPTION
**Pull request by rtyler**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -9,12 +9,16 @@ image::https://badges.gitter.im/Join%20Chat.svg[link="https://gitter.im/jruby-gr
 
 You can also join us on the link:https://groups.google.com/forum/#!forum/jruby-gradle[JRuby/Gradle mailing list]
 
-The purpose of these plugins is to bundle useful link:http://gradle.org[Gradle]
-functionality for JRuby projects. Use of this plugin replaces the need for both link:http://bundler.io[Bundler] and link:https://github.com/jruby/warbler[Warbler].
+JRuby/Gradle brings the power and flexibilitye of
```
Comment (ysb33r) [inline README.adoc]: *flexibility*

File: README.adoc [hunk 2]
```diff
@@ -9,12 +9,16 @@ image::https://badges.gitter.im/Join%20Chat.svg[link="https://gitter.im/jruby-gr
 
 You can also join us on the link:https://groups.google.com/forum/#!forum/jruby-gradle[JRuby/Gradle mailing list]
 
-The purpose of these plugins is to bundle useful link:http://gradle.org[Gradle]
-functionality for JRuby projects. Use of this plugin replaces the need for both link:http://bundler.io[Bundler] and link:https://github.com/jruby/warbler[Warbler].
+JRuby/Gradle brings the power and flexibilitye of 
+link:http://gradle.org[Gradle]
```
Comment (ysb33r) [inline README.adoc]: You can probablyhave everything on a single line. This is pretty much Asciidoc writing convention. Most editors can wrap long lines and INtelliJ actually has a setting which does this automatically for`.doc` files.

File: docs/errors/jar-deprecated-jrubyversion.adoc [hunk 3]
```diff
@@ -10,6 +10,9 @@ JRuby in an "embedded" scenario more reliable. Packing a `JRubyJar` being a
 fairly typical "embedded JRuby" use-case, there is certain functionality that
 relies on these improvements.
 
+
+NOTE: With JRuby/Gradle 2.0.0, JRuby versions 1.7.x are **completely** unsupported.
```
Comment (ysb33r) [inline docs/errors/jar-deprecated-jrubyversion.adoc]: I would rewrite this as: *As from JRuby-Gradle 2.0, the JRuby 1.7.x releases are no longer supported.*

File: docs/index.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ link:http://jruby.org[JRuby] with Gradle's excellent task and dependency
 management, *JRuby/Gradle* provides high quality build tooling for
 Ruby and Java developers alike.
 
-NOTE: As from version 1.6.0 you will need at last Gradle 2.8.
+NOTE: As from version 2.0.0 you will need at least Gradle 4.2.
```
Comment (ysb33r) [inline docs/index.adoc]: :+1:

File: README.adoc (hunk 2 above)
Comment (rtyler) [reply README.adoc]: This is a style pattern we use in the Jenkins documentation to make life a little bit easier, putting links on their own lines

**Comment by rtyler** [comment]
I'll take a look at the test failures and tidy up the text later today on the commute home :smile_cat:

**Comment by rtyler** [comment]
Travis is green, merging away! :tada:

**Pull request merged**